bin/strata-cli/src/constants.rs

```diff
@@ -37,3 +37,28 @@ pub const SIGNET_BLOCK_TIME: Duration = Duration::from_secs(30);
 
 pub const BRIDGE_MUSIG2_PUBKEY: &str =
     "14ced579c6a92533fa68ccc16da93b41073993cfc6cc982320645d8e9a63ee65";
+
+/// BIP44 purpose index for HD wallets.
+///
+/// These should be _hardened_ [`ChildNumber`].
+pub const BIP44_HD_WALLET_IDX: u32 = 44;
+
+/// BIP44 coin type index to indicate Testnet.
```

Comment:
```suggestions
/// BIP44 coin type index to indicate EVM Mainnet.
```
60 actually indicates mainnet, which we also use for the testnet (as others also do, and we want to remain compatible on the wallet level).